app/src/main/java/to/dev/dev_android/view/main/view/MainActivity.kt

```diff
@@ -43,6 +45,7 @@ class MainActivity : BaseActivity<ActivityMainBinding>(), CustomWebChromeClient.
     private fun setWebViewSettings() {
         binding.webView.settings.javaScriptEnabled = true
         binding.webView.settings.domStorageEnabled = true
+        binding.webView.addJavascriptInterface(this.webViewBridge, "androidWebViewBridge")
```

Comment:
We don't actually need the `this` to pass the parameter. I would say the less verbose, the readable.